composer.json

```diff
@@ -19,7 +19,8 @@
         "php": ">=5.3.9",
         "doctrine/common": "~2.3",
         "twig/twig": "~1.18",
-        "psr/log": "~1.0"
+        "psr/log": "~1.0",
+        "symfony/templating": "~2.8"
```

Comment:
The dependency [already exists in the FrameworkBundle](https://github.com/symfony/symfony/blob/2.8/src/Symfony/Bundle/FrameworkBundle/composer.json#L31), which is ok. We need to add this dependency here to keep BC, so, if someone requires the `symfony/symfony:~2.8` package can use the Templating component without any extra change in the `composer.json` file.